library/std/src/keyword_docs.rs

```diff
@@ -2178,7 +2166,7 @@ mod where_keyword {}
 ///
 /// It is available for use in stable rust from version 1.39 onwards.
 ///
-/// [`Future`]: ./future/trait.Future.html
+/// [`Future`]: future::Future
```

Comment:
`future` is not in scope.
```suggestion
/// [`Future`]: crate::future::Future
```